config.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"github.com/jawher/mow.cli"
+)
+
+type serviceConfig struct {
+	appSystemCode            *string
```

Comment:
I prefer using simple string types, instead of pointers - if there is no particular reason for using pointers. Same for the createServiceConfiguration functions' return type.